panelapp/management/tests/import_all_panels_tests.py

```diff
@@ -0,0 +1,17 @@
+import mock
+
+from django.core.management import call_command
+from django.test import TestCase
+
+
+class ImportAllPanelsTest(TestCase):
```

Comment:
I don't think you need this test file at all - in test_import_all_panels in the pa_locus_list_api_tests file you test this in much greater depth so I don't think this file really adds anything